rackcity/views/pdu_views.py

```diff
@@ -15,23 +15,25 @@
     log_power_action,
     PowerAction,
 )
+from rackcity.permissions.decorators import power_permission_required
 from rest_framework.parsers import JSONParser
+from rest_framework.permissions import IsAuthenticated
 from rest_framework.decorators import permission_classes, api_view
-from rest_framework.permissions import IsAuthenticated, IsAdminUser
 from http import HTTPStatus
 import re
 import requests
 import time
 from requests.exceptions import ConnectionError
 
+
 pdu_url = 'http://hyposoft-mgt.colab.duke.edu:8005/'
 # Need to specify rack + side in request, e.g. for A1 left, use A01L
 get_pdu = 'pdu.php?pdu=hpdu-rtp1-'
 toggle_pdu = 'power.php'
 
 
 @api_view(['GET'])
-@permission_classes([IsAuthenticated])
+@power_permission_required()
```

Comment:
Is power permission required to see the power status of an asset? I didn't think it was

Reply:
oh you're right I was wrong. fixing!